tests/services/format_trade_service_spec.coffee

```diff
@@ -0,0 +1,54 @@
+describe "formatTrade service", () ->
+	formatTrade = undefined
+	$rootScope = undefined
+
+	beforeEach angular.mock.module("walletApp")
+
+	beforeEach ->
+		angular.mock.inject ($injector, _$rootScope_, _$q_) ->
+			$rootScope = _$rootScope_
+			$q = _$q_
+			Wallet = $injector.get("Wallet")
+
+			formatTrade = $injector.get("formatTrade")
+
+			spyOn(formatTrade, "checkDKK").and.callThrough()
+
+	trade = (currency) ->
```

Comment:
Good idea making this a function instead of an object, easier to use in future tests that way.